apollo-router/src/configuration/mod.rs

```diff
@@ -603,6 +613,53 @@ pub(crate) struct RedisCache {
     pub(crate) urls: Vec<String>,
 }
 
+/// Configuration options pertaining to the subgraph server component.
+#[derive(Debug, Clone, Deserialize, Serialize, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct Tls {
+    #[serde(default)]
+    pub(crate) subgraphs: Option<Subgraph>,
+}
+
+#[buildstructor::buildstructor]
+impl Tls {
+    #[builder]
+    pub(crate) fn new(subgraphs: Option<Subgraph>) -> Self {
+        Self { subgraphs }
+    }
+}
```

Comment:
honestly I added the builder mainly because there were builders for other parts of the configuration :sweat_smile: 
It's not strictly necessary right now, but it's better to have a consistent API, no?

Reply:
Yeah, if we change that it should probably in its own PR, for all config structs at once